groups/rds/variables.tf

```diff
@@ -16,7 +16,7 @@ variable "default_ami_version_pattern" {
 
 variable "default_instance_type" {
   description = "The default instance type to use for instances"
-  default     = "t3.medium"
+  default     = "t3.large"
   type        = string
 }
```

Comment:
This variable isn't actually used in the `rds` group and so can be removed entirely.

Reply:
Thanks @sienkin. Variable has now been removed